### DISCUSSION TITLE
Which databases should DbGate support?

### DISCUSSION THREAD

**Discussion by janproch** [dbgate/dbgate#212] [Ideas]
You could express your wish, what will be the next supported database in DbGate. I can't promise, that I will choose database with most votes, but it will be important criteria.

If you don't see your favorite database here and it is not yet suported in DbGate, let here comment about it. Or create new issue.

[![](https://api.gh-polls.com/poll/01FTZ6H7XKDHCWJ2D91JS70FKM/Oracle)](https://api.gh-polls.com/poll/01FTZ6H7XKDHCWJ2D91JS70FKM/Oracle/vote)
[![](https://api.gh-polls.com/poll/01FTZ6H7XKDHCWJ2D91JS70FKM/ClickHouse)](https://api.gh-polls.com/poll/01FTZ6H7XKDHCWJ2D91JS70FKM/ClickHouse/vote)
[![](https://api.gh-polls.com/poll/01FTZ6H7XKDHCWJ2D91JS70FKM/Redis)](https://api.gh-polls.com/poll/01FTZ6H7XKDHCWJ2D91JS70FKM/Redis/vote)
[![](https://api.gh-polls.com/poll/01FTZ6H7XKDHCWJ2D91JS70FKM/Firebird)](https://api.gh-polls.com/poll/01FTZ6H7XKDHCWJ2D91JS70FKM/Firebird/vote)
[![](https://api.gh-polls.com/poll/01FTZ6H7XKDHCWJ2D91JS70FKM/Mindsdb)](https://api.gh-polls.com/poll/01FTZ6H7XKDHCWJ2D91JS70FKM/Mindsdb/vote)

**Comment by kyleli666** [comment]
DolphinDB: https://dolphindb.com/

BTW, DbGate is the best database tool to me.

**Reply by ProjectInfinity** [reply]
It says SQL ANSI compatible, but it probably needs a separate driver. It does not seem like the existing SQL drivers would handle it.

**Reply by kyleli666** [reply]
There are some API available, hope they are helpful.
https://dolphindb.com/product/development-resources?default_type=api_resource
One of them is the javascript api https://github.com/dolphindb/api-javascript/blob/main/README.md

**Comment by nathan-g-2degrees** [comment]
Would be great to see support for Snowflake implemented, there is an official pure JavaScript driver for Snowflake https://docs.snowflake.com/en/developer-guide/node-js/nodejs-driver

**Comment by Jengro777** [comment]
Support TiDB Serverless

https://tidbcloud.com/

**Comment by alalolu** [comment]
Hope it would support Teradata DB
https://downloads.teradata.com/download/connectivity/jdbc-driver

**Comment by mquan86** [comment]
Support H2 database would be great. Thanks.

(https://www.h2database.com/html/main.html)
